samples/OpenIdConnectSample/Startup.cs

```diff
@@ -17,7 +17,7 @@ public void Configure(IApplicationBuilder app)
                 services.AddDataProtection();
                 services.Configure<ExternalAuthenticationOptions>(options =>
                 {
-                    options.SignInAsAuthenticationType = CookieAuthenticationDefaults.AuthenticationType;
+                    options.SignInAsAuthenticationScheme = CookieAuthenticationDefaults.AuthenticationScheme;
```

Comment:
*shrug* I think the verbose version is fine, it won't come up in user code much.